features/show_membership_application.feature

```diff
@@ -86,3 +86,14 @@ Feature: As an Admin
     Given I am logged in as "hans@random.com"
     And I am on the list applications page
     Then I should see "Du har inte behörighet att göra detta."
+
+  Scenario: Clicking the edit button on show page
+    Given I am logged in as "admin@sgf.com"
+    When I am on the application page for "Emma"
+    And I click on "Redigera ansökan"
+    Then I should be on the edit application page for "Emma"
+
+  Scenario: User does not see edit-link
+    Given I am logged in as "emma@random.com"
+    When I am on the application page for "Emma"
```

Comment:
Why not use the step that already exists:  instead of using the first name ("Emma"), just look up the page by the email.
`When I am on the application page for "emma@random.com`

It's faster (one less db hit), and then the added step (below) is not necessary.